src/main/java/sg/ncl/MainController.java

```diff
@@ -746,6 +747,7 @@ public String validateDetails(
             }
 
             teamFields.put("id", joinTeamInfo.getId());
+            teamFields.put("notes", signUpMergedForm.getJoinTeamReason());
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "notes" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)